sympy/series/tests/test_series.py

```diff
@@ -202,6 +202,14 @@ def test_issue_9549():
     assert series(y, x, oo) == x**(-5) - 1/x**4 + x**(-3) + 1/x + O(x**(-6), (x, oo))
 
 
+def test_issue_9173():
+    from sympy import var
```

Comment:
Imports should go on top of file.

Reply:
Ok I will do it after these checks will be completed.

Reply:
And `var` should not be used in library code.

Reply:
Ok thank you @jksuom ,so can you pease suggest me what to use instead of var or simply where I will find the rules related to uses of variables.

Reply:
As used in other tests like `test_issue_12791`, you can use `symbols` instead

Reply:
As used in other tests like `test_issue_12791`, you can use `symbols` instead